### DISCUSSION TITLE
attachment.get()

### DISCUSSION THREAD

**Discussion by acckiydarik** [pycontribs/jira#1142] [General]
Hello, I couldn't find an option to print content of only one particular file by name or id from attachments.
Is it possible?
https://jira.readthedocs.io/en/master/examples.html#attachments
thank you.

**Comment by adehad** [comment]
The example shows how you can iterate through available attachments, so if you know the name of the attachment you want, you can check that before printing.

For example

```python
ATTACHMENT_NAME = "expected_file.txt"

for attachment in issue.fields.attachment:
    if attachment.filename != ATTACHMENT_NAME:
        continue # continue for loop if the filename does not match
    print("Name: '{filename}', size: {size}".format(
        filename=attachment.filename, size=attachment.size))
    # to read content use `get` method:
    print("Content: '{}'".format(attachment.get()))

```
